server/util/errorLogger.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
 			})
 		],
 		meta: false,
-		requestWhitelist: ['url', 'responseTime', 'level', 'message', 'headers.host', 'headers.method'],
```

Comment:
This is part of the winston API. They are working on changing these terms for the upcoming 5.0 release, but we cannot make this modification just yet....

Reply:
Thanks @eddieferrer for catching this!